packages/demo/usecase-demo/src/modules/createNew/CreateNewContent.tsx

```diff
@@ -137,7 +137,7 @@ export function CreateNewContent() {
       toast({
         variant: 'danger',
         title: 'File is too big',
-        description: `Selected file is ${Math.round(fileSizeInKb)} Kb, should be less than ${FILE_SIZE_LIMIT_IN_KB} Kb.`,
+        description: `Selected file is ${Math.round(fileSizeInKb / 1000)} Mb, should be less than ${FILE_SIZE_LIMIT_IN_KB / 1000} Mb.`,
```

Comment:
When selected file is too big, easier to have values in Mb instead of Kb in this error message.

<img width="694" alt="Capture d’écran 2024-07-23 à 12 02 05" src="https://github.com/user-attachments/assets/4152c99d-68b7-4c59-8ba5-311dee03b8f4">